R/set_args.R

```diff
@@ -54,6 +54,23 @@ set_legend_args <- function(object, fun.name, ..., legend.name) {
   
   paramsAll <- set_args(fun.name, list(...))
   
+  if(length(legend.name) > 1){
+    paramsAll_df <- as.data.frame(paramsAll, stringsAsFactors = FALSE)
```

Comment:
I would like some input on a more elegant way to accomplish this - trying to separate the list of params into multiple lists of params. E.g. if `listAll <- list(x=c(1,2), y=c(2,3), col=c("red", "green"), legend.name = c("point 1", "point 2"))` then I would like to be able to send `list1 <- list(x=1, y=2, col="red", legend.name="point 1")` and `list2 <- list(x=2, y=3, col="green", legend.name="point 2")` to `get_legend_args` separately.